kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -0,0 +1,56 @@
+www_root="/var/www"
+app_root="$www_root/miq"
+appliance_root="/opt/manageiq/manageiq-appliance"
+
+rm -rf $app_root
+
+git clone <%= @appliance_checkout.remote %> $appliance_root
+pushd $appliance_root
+  git checkout <%= @appliance_checkout.branch %>
+  git reset --hard <%= @appliance_checkout.commit_sha %>
+popd
+
+mkdir -p $app_root
+git clone <%= @manageiq_checkout.remote %> $app_root/vmdb
+pushd $app_root/vmdb
+  git checkout <%= @manageiq_checkout.branch %>
+  git reset --hard <%= @manageiq_checkout.commit_sha %>
+popd
+
+# Symlink extracted repo (manageiq-appliance) to old /var/www/miq/system location.
+ln -vs $appliance_root $app_root/system
+
+# Symlink old lib directory to gems/pending.
+ln -vs $app_root/vmdb/gems/pending $app_root/lib
+
+#### TODO: Refactor this so the cfme rpm and the upstream share much of this code.
+
+mkdir -p $app_root/vmdb/log/apache
+mkdir -p /etc/httpd/conf.d
+
+pushd $appliance_root
+  #symlink some executables
+  mkdir -p /bin
+  pushd ./LINK/bin
+    for filename in `ls`; do
+      ln -vs $appliance_root/LINK/bin/$filename /bin/$filename
+    done
+  popd
+
+  #symlink some configuration files
+  pushd ./LINK/etc
+    for dirname in `ls`; do
+      pushd ./$dirname
+        mkdir -p /etc/$dirname
+        for filename in `ls`; do
+          ln -vs $appliance_root/LINK/etc/$dirname/$filename /etc/$dirname/$filename
+        done
+      popd
+    done
+  popd
+
+  ln -vs $appliance_root/LINK/.toprc /.toprc
+
+  #copy all files/directories below COPY
+  cp -vr COPY/* /
+popd
```

Comment:
Same issue as on line 25.  I wouldn't want to move the code into a partial other than where the variables are defined.